test/tests/concrete_drying_shrinkage/tests

```diff
@@ -7,23 +7,23 @@
     [shrinkage]
       type = 'Exodiff'
       input = 'shrinkage.i'
-#      abs_zero = 1e-5
+      abs_zero = 2e-5
```

Comment:
This file shouldn't be changed. There was another commit that got merged in that modified these.